src/openrct2/scripting/ScEntity.hpp

```diff
@@ -592,6 +607,18 @@ namespace OpenRCT2::Scripting
             }
             return result;
         }
+
+        void move(int32_t value)
+        {
+            ThrowIfGameStateNotMutable();
+            auto vehicle = GetVehicle();
+            if (vehicle != nullptr)
+            {
+                vehicle->Invalidate2();
```

Comment:
I'm pretty sure `MoveRelativeDistance` will now take care of the invalidating.

Reply:
Yes you're right, I'll remove it. :)